lib/screens/home_screen.dart

```diff
@@ -142,31 +151,29 @@ class _HomeScreenState extends State<HomeScreen> {
               ),
               ScreenResponsive(
                 mobileScreen: (context) => const VerticalSpace(6),
-                tabletScreen: (BuildContext context) => const VerticalSpace(30),
+                tabletScreen: (context) => const VerticalSpace(30),
               ),
               Expanded(
                 child: PageView(
                   controller: _pageController,
                   physics: const NeverScrollableScrollPhysics(),
                   onPageChanged: (int page) {
                     _currentPage.value = page;
+                    if (page <= 2) {
```

Comment:
constants "2" and "3" can be in variable.
switch/case might be more efficient here.